app/pb_kits/playbook/pb_textarea/_textarea.html.erb

```diff
@@ -15,10 +15,14 @@
         :object,
         :method,
```

Comment:
The first two options in this `text_area` helper method are hard coded will not work. They need to be `object.object` and `object.method`.